system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -321,6 +323,11 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 parameter_types! {
 	// Fellows pluralistic body.
 	pub const FellowsBodyId: BodyId = BodyId::Technical;
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = Concrete(xcm_config::KsmRelayLocation::get());
+	/// The base fee for the message delivery fees.
+	pub const ToSiblingBaseDeliveryFee: u128 = CENTS.saturating_mul(3); // TODO: proper price
```

Comment:
That's the price we have set on testnets. But maybe we need to change it for Polkadot, Kusama and system parachains?